src/test/java/io/candydoc/ddd/tactical_patterns/domain_entity/DomainEntityRulesTest.java

```diff
@@ -0,0 +1,109 @@
+package io.candydoc.ddd.tactical_patterns.domain_entity;
+
+import static io.candydoc.ddd.AssertDependency.assertThatDependency;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.candydoc.ddd.model.*;
+import io.candydoc.ddd.tactical_pattern.DomainEntity;
+import io.candydoc.ddd.tactical_pattern.TacticalPattern;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class DomainEntityRulesTest {
+  static DomainModel entity;
+
+  @BeforeAll
+  static void setUp() {
+    entity =
+        DomainEntity.builder()
+            .fullName(FullName.of("io.candydoc.sample.EntityOne"))
+            .simpleName(SimpleName.of("EntityOne"))
+            .packageName(PackageName.of("io.candydoc.sample"))
+            .dependency(
+                Dependency.builder()
+                    .name(FullName.of("io.candydoc.sample.ValueObjectOne"))
+                    .pattern(TacticalPattern.VALUE_OBJECT)
+                    .build())
+            .dependency(
+                Dependency.builder()
+                    .name(FullName.of("io.candydoc.sample.DomainCommandOne"))
+                    .pattern(TacticalPattern.DOMAIN_COMMAND)
+                    .build())
+            .dependency(
+                Dependency.builder()
+                    .name(FullName.of("io.candydoc.sample.DomainEventOne"))
+                    .pattern(TacticalPattern.DOMAIN_EVENT)
+                    .build())
+            .dependency(
+                Dependency.builder()
+                    .name(FullName.of("io.candydoc.sample.AggregateOne"))
+                    .pattern(TacticalPattern.AGGREGATE)
+                    .build())
+            .dependency(
+                Dependency.builder()
+                    .name(FullName.of("io.candydoc.sample.EntityTwo"))
+                    .pattern(TacticalPattern.DOMAIN_ENTITY)
+                    .build())
+            .build();
+  }
+
+  @Test
+  void is_a_domain_entity() {
+    assertThat(entity.dddPattern()).isEqualTo(TacticalPattern.DOMAIN_ENTITY);
+  }
+
+  @Test
```

Comment:
As we keep adding the new concepts, I think our tests `it_can_depend` and `it_can_not_depend` should be improved to handle a list of all allowed and disallowed concepts. We could do it through Parameterized Test and a source of Tactical / Strategic Patterns. IMO, we should do it on a specific PR to do this only one time for each concept

Reply:
I see that you started to do something like that in this PR: https://github.com/Candy-Doc/candy-doc-maven-plugin/pull/195#pullrequestreview-1467391975

Reply:
It's a complex one. I mean, it's hard to retrieve the list of allowed_dependencies without using the _ALLOWED_DEPENDENCIES_ field. If we use it anyway, It'll look like the snake is bitting his own tail.

Reply:
Actually, it's doing well with _@repository_, given that just the aggregate is allowed as dependency.